front/src/applications/stdcmV2/components/StdcmConsist.tsx

```diff
@@ -93,26 +93,8 @@ const StdcmConsist = ({ setCurrentSimulationInputs, disabled = false }: StdcmCon
     } else {
       searchRollingStock('');
     }
-    setCurrentSimulationInputs((prevState) => ({
-      ...prevState,
-      consist: {
-        tractionEngine: rollingStock,
-      },
```

Comment:
Previously, changing the rolling stock would clear any previously set `speedLimitByTag`. Do we want to retain this behavior?

If so, we need to call `dispatchUpdateSpeedLimitByTag()` in the spots where we're calling `updateRollingStockID()`.

Reply:
You're right, I kept the previous behavior :+1: